packages/examples/packages/wasm/build/program.d.ts

```diff
@@ -0,0 +1,13 @@
+declare namespace __AdaptedExports {
```

Comment:
This file shouldn't be committed. Is this folder not in the `.gitignore`?

Reply:
Something must have gone awry when I merged, I had to clear the cache.